build/Linux/build/common/run.sh

```diff
@@ -2,4 +2,4 @@
 
 # This script can be used to run a dotnet application with New Relic monitoring.
 
-CORECLR_NEW_RELIC_HOME=${CORECLR_NEW_RELIC_HOME:-/usr/local/newrelic-dotnet-agent} CORECLR_ENABLE_PROFILING=1 CORECLR_PROFILER={36032161-FFC0-4B61-B559-F6C5D41BAE5A} CORECLR_PROFILER_PATH=$CORECLR_NEW_RELIC_HOME/libNewRelicProfiler.so $@
+NRHOME=${CORECLR_NEWRELIC_HOME:-${CORECLR_NEW_RELIC_HOME:-/usr/local/newrelic-dotnet-agent}} CORECLR_ENABLE_PROFILING=1 CORECLR_PROFILER={36032161-FFC0-4B61-B559-F6C5D41BAE5A} CORECLR_PROFILER_PATH=$NRHOME/libNewRelicProfiler.so $@
```

Comment:
This is intentional; uses either the "old" or "new" environment variable, whichever is found first

Reply:
Unfortunately, this doesn't work because AgentInstallConfiguration.cs doesn't look for NRHOME, so if neither CORECLR_NEWRELIC_HOME or CORECLR_NEW_RELIC_HOME is set because the application is depending on run.sh to set these, the profiler/agent ends up not running. See #2887